modules/publication/help/publication.md

```diff
@@ -0,0 +1,48 @@
+# Publication
+
+The publication module allows users to propose research projects based on LORIS 
+data, upload publication media, and allows study PIs to review the project 
+proposals.
+
+## Intended Users
```

Comment:
The help directory is for user-facing help that gets loaded when clicking the "?" icon on the front end which should explain how to use the module. This seems to be specifications for the README.md, not help text.